src/pages/Main/MainPage.js

```diff
@@ -35,7 +35,7 @@ function MainPage(props) {
         loadMessageData(messageData);
 
         changeCurrentGroup(1, groupData[indexGroup].groupName); //default group is first group in list
-    }, []);
+    }, [loadGroupData, loadMessageData, changeCurrentGroup]);
```

Comment:
why????

Reply:
either that, this `eslint-disable-line react-hooks/exhaustive-deps` or apply `useCallback` because react throws warning attached below when CI is building project.
What best way to to it?
![image](https://user-images.githubusercontent.com/86774452/129531373-9d88fe41-42e2-4956-852c-b0fba7d4144f.png)

Reply:
> note : I need dependencies array in useEffect to render only once, otherwise there will be infinite loop.

Reply:
It not right subscribe on methods
if you need dependences from array, made this from array not from actions